app/models/katello/glue/pulp/repo.rb

```diff
@@ -199,7 +199,7 @@ def importer_ssl_options(capsule = nil)
             :ssl_ca_cert => nil
           }
         end
-        importer_options.merge!(:ssl_validation => verify_ssl_on_sync?) unless self.is_a?(::Katello::ContentViewPuppetEnvironment)
+        importer_options.merge(:ssl_validation => verify_ssl_on_sync?) unless self.is_a?(::Katello::ContentViewPuppetEnvironment)
```

Comment:
It does not, I had added a commit by John Mitsch to make some of this work awhile ago, and had not rebased properly. About to remove that and have it working properly.